test/spec/cloud-element-templates/properties/NumberProperty.spec.js

```diff
@@ -61,6 +61,44 @@ describe('provider/cloud-element-templates - NumberProperty', function() {
   }));
 
 
+  describe('property', function() {
+    let entry, input;
+
+    beforeEach(async function() {
```

Comment:
I'd ensure that we also have tests to verify this against the element templates API, applying a template.

Reply:
Done via https://github.com/bpmn-io/bpmn-js-element-templates/pull/161/commits/7e282c92fb76fe92da651872fc100b8280fca92b